tests/index.js

```
@@ -1,169 +1,183 @@
+    const newUser = {
+      name: "Keber",
+      age: 24,
+      hobbies: ["dubstep"],
+    };
+
+    const res = await axios.post("http://localhost:9000/person", newUser);
+    assert.equal(res.status, 200);
+
+    const persons = app.get("db");
+    const insertedUser = Object.assign({}, persons[1]);
+    delete insertedUser.id;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)